src/Handler/SentryMonologHandler.php

```diff
@@ -9,55 +9,55 @@
 
 namespace PhpTek\Sentry\Handler;
 
-use PhpTek\Sentry\Monolog\Handler\SentryRavenHandler,
-    Monolog\Logger,
-    SilverStripe\Dev\Backtrace,
-    SilverStripe\Security\Member,
-    PhpTek\Sentry\Log\SentryLogger;
+use Exception;
+use Monolog\Logger;
+use PhpTek\Sentry\Log\SentryLogger;
+use PhpTek\Sentry\Monolog\Handler\SentryRavenHandler;
+use SilverStripe\Dev\Backtrace;
+use SilverStripe\Security\Member;
 
 /**
  * Monolog Handler for Sentry via Raven
  */
-
 class SentryMonologHandler extends SentryRavenHandler
-{    
+{
     /**
      * @var SentryClientAdaptor
      */
     protected $client;
 
     /**
-     * @param  int   $level
-     * @param  bool  $bubble
```

Comment:
Which linting tool / ruleset are you using? I rather like vertically aligned param descriptions.

Reply:
PHPStorm auto-format, sorry. I can undo if you like.

Reply:
in Editor > Code Style > PHP

I've turned this on. :)

![image](https://user-images.githubusercontent.com/936064/49549638-35ca0f00-f94e-11e8-91f6-89e74a85b85a.png)